packages/big-design/src/components/TableNext/Row/Row.tsx

```diff
@@ -165,7 +165,7 @@ const InternalRow = <T extends TableItem>({
               verticalAlign={verticalAlign}
               width={isDragging ? cellWidth : width}
             >
-              <Box display="flex">
+              <Flex alignItems="center">
```

Comment:
Do we also need `flexDirection="row"`? On mobile it automatically switches to `column`